Source/WebServices/UIImageView+SDExtensions.h

```diff
@@ -79,6 +79,11 @@ typedef enum {
 - (void)cancelCurrentImageLoad;
 
 /**
+ * Return Image URL
+ */
+- (NSURL *)imageURL;
```

Comment:
Doh, totally missed that property - thanks, will kill this PR (and the release branch)